### PR TITLE
add JBT 4.2.3.CR1 and link to blog >> master

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

File: _config/products.yml [hunk 1]
```diff
@@ -439,6 +439,34 @@ jbt_core:
...
+          - name: Update site (including sources) bundle of all Hibernate Tools
+            file_size: 105MB
+            url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.3.x/jbosstools-4.2.3.CR1_2015-03-14_08-49-30-B257-updatesite-hibernatetools.zip.MD5
+            md5_url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.3.x/jbosstools-4.2.3.CR1_2015-03-14_08-49-30-B257-updatesite-hibernatetools.zip.MD5
+          - name: Stand-alone BrowserSim
+            file_size: 15.8MB
+            url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.3.x/jbosstools-4.2.3.Beta1_2015-03-01_13-40-40-B357-browsersim-standalone.zip
+            md5_url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.3.x/jbosstools-4.2.3.Beta1_2015-03-01_13-40-40-B357-browsersim-standalone.zip.MD5
+          - name: Release Notes
+            url: http://tools.jboss.org/documentation/whatsnew/jbosstools/4.2.3.Beta1.html
```
Comment (alexeykazakov) [inline _config/products.yml]: Should be CR1

File: _config/products.yml [hunk 2]
```diff
@@ -439,6 +439,34 @@ jbt_core:
...
 
     luna:
+      4.2.3.CR1:
+        update_site_url: http://download.jboss.org/jbosstools/updates/development/luna/
+        #marketplace_url: http://marketplace.eclipse.org/node/1617241
+        #marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1617241
+        #archived: true
+        release_date: 2015-03-19
+        supported_jbt_is_version: 4.2.0.Final
+        blog_announcement_url: /blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.html
```
Comment (maxandersen) [inline _config/products.yml]: when fixed in blog this shuold be updated to -devstudio- too ?

Comment (nickboldt) [inline _config/products.yml]: Should we also fix the Beta1 blog then? https://github.com/nickboldt/jbosstools-website/blob/release423CR1/_config/products.yml#L70

**Comment by alexeykazakov** [comment]
Pushed to master and production.

**Pull request closed without merging**